conf.yaml

```diff
@@ -591,3 +606,8 @@ targets:
        - vapiano.com
        - de.vapiano.com
     twitter: '@vapiano'
+  WIRED:
+    href: https://www.wired.com
+    hosts:
+      - www.wired.com
```

Comment:
also `wired.com`, it has some more hosts.

Reply:
I don't understand why. Connecting to `https://wired.com` returns a HTTP 301 to `https://www.wired.com`. I don't think "we" should test redirects. That would be sooo much work to check for all targets.

```
:-$ curl --head https://wired.com
HTTP/2 301
server: Varnish
[...]
location: https://www.wired.com/
[...]
```

Reply:
OK.